airbyte-integrations/connectors/source-shopify/source_shopify/spec.json

```diff
@@ -127,16 +132,6 @@
             "path_in_connector_config": ["credentials", "client_secret"]
           }
         }
-      },
-      "oauth_user_input_from_connector_config_specification": {
```

Comment:
for my understanding why isn't think a breaking change? Is it because `oauth_user_input_from_connector_config_specification` isn't used anywhere in the code?

Reply:
The `breaking change` was considered before your review, as we decided to leave both options, there is no breaking change for now. Once everybody is successfully migrated to the `OAuth2.0`  we will manage the Cloud auth options with `spec handler` and that's it.

The `oauth_user_input_from_connector_config_specification` is part of the `advancedAuth` and not used anywhere in the code. As far as we automatically push the `shop` from the confirmation URL after the successful installation of the Airbyte<>Shopify public app, we no longer need to pass it manually for the OAuth Flow. Therefore it's removed.

Reply:
sounds good, thanks for clarifying!